docs/Reference/ID-Prefix-Reservation.md

```diff
@@ -0,0 +1,154 @@
+*  During prefix reservation, the owner can choose to make a prefix public. This will still give them the visual indicator showing that the package originates from a reserved prefix, but it will **not** block future package submissions on the prefix for any owner. This is useful for open source projects with many contributors - the top or core contributors can have the prefix reserved, but it can still be open to all contributors. 
+
+### Prefix resveration visual indicator
+When a package comes from a reserved prefix, you will see the below visual indicators on the [NuGet.org](https://www.nuget.org/) gallery and in Visual Studio 2017 version 15.4 or later:
+
+**NuGet.org Gallery**
+![NuGet.org Gallery](media/nuget-gallery-reserved-prefix.png)
+
+**Visual Studio**
+![Visual Studio](media/visual-studio-reserved-prefix.png)
+
+## ID prefix reservation application process
+To apply for a prefix reservation, follow the below steps. 
+1. Review the acceptance [criteria for prefix ID reservation](#id-prefix-reservation-criteria).
+2. Determine the namespaces you want to reserve, in addition to any [advanced prefix reservation scenarios](#advanced-prefix-reservation-scenarios) you may require.
+3. Send a mail to [account@nuget.org](mailto:account@nuget.org) with the owner display name on [NuGet.org](https://www.nuget.org/), as well as any reserved prefixes you are requesting. If you are delegating prefix subsets to multiple owners, make sure you mention all owner display names and prefix subsets.
+
+After the application is submitted, you will be notified of acceptance or rejection (with the criteria that caused rejection). We may need to ask additional identifying questions to confirm owner identity. 
+
+### ID prefix reservation criteria
+When reviewing any application for ID prefix reservation, the [NuGet.org](https://www.nuget.org/) team will evalaute the application against the below criteria. Not all criteria needs to be met for a prefix to be reserved, but the application may be denied if there is not substantial evidence of the criteria being met (with an explanation given):
+1. Does the package ID prefix properly and clearly identify the package owner?
+2. Are a significant number of the packages that have already been submitted by the owner under the package ID prefix?
+3. Is the package ID prefix something common that should not belong to any individual owner or organization?
+4. Would *not* reserving the package ID prefix cause ambiguity and confusion for the community?
+5. Are the identifying properties of the packages that match the package ID prefix clear and consistent (especially the package author)?
+
+## 3rd party feed provider scenarios
```

Comment:
what about the guidelines to allocate namespace reservation? Shouldn't we suggest some key points for a 3rd party feed to assign/allocate a reserved prefix, similar adherence to our guidelines or something like that?

Reply:
What do you have in mind? Independent feed providers can choose how they want to use the feature, such as reserving for specific teams, reserving for internally created packages, or similar?